eaw-functions-emmyluadoc.lua

```diff
@@ -0,0 +1,81 @@
+--######################################################################################
+--############################## Finder Functions ######################################
+--######################################################################################
+---@public
+---@param type_name string A type name as specified in the XML code
+---@return GameObject|nil
+---Returns the first GameObject of the given type name
+function Find_First_Object(type_name)
+end
+
+---@public
+---@param type_or_category string A type or category name as specified in the XML code
+---@param filter PlayerObject|string PlayerObject or CategoryMask
+---@return GameObject[]
```

Comment:
Does that syntax work with Emmy? I thought it had to be `table<GameObject>`?
I haven't really looked into the latest changes too much though.